src/components/pages/DAOcreator/SchemesStep/AddSchemeDialog.tsx

```diff
@@ -95,6 +99,19 @@ class VerticalLinearStepper extends React.Component<Props, State> {
     }))
   }
 
+  handleSchemeConfigParamsChange = async (event: any) => {
+    const { name, value } = event.target
+    if (this.state.schemeConfig != null) {
+      await this.setState({
+        schemeConfig: {
+          id: this.state.schemeTypeName,
```

Comment:
should this be a `uuid()`

Reply:
Yes, because there can be multiple schemes of each type